### ISSUE TITLE
Ability to use more than 8 colour palettes in dialogue boxes.

### ISSUE THREAD

**Issue by chrcoluk** [myst6re/makoureactor#145]
The game supports more than 8 colour palettes   They just not populated by default, if support was added in makou reactor it would mean support extra icons to be used in field text such as mini avatars or controller icons would be easier, is it possible to implement a variable for them if one doesnt already exist?

**Comment by myst6re** [comment]
Sorry I'm not sure to follow what you ask, also I am not aware of this "more than 8 colours" thing in usfont. Can you elaborate?

**Comment by chrcoluk** [comment]
Yeah sure, there is the palette id at end of each menu texture e.g. usfont_a_00
So e.g. 02 is blue and 04 is red, the existing colours use 0,2,4,6,8,10,12,14.

But unused is odd numbers like 1,3,5,7,9,11 and 13.

I was hoping if its possible to make it so any symbol or character typed into makou reactor could be made to use any of those odd numbered palettes.

**Comment by Tsunamods** [comment]
Sounds like a good idea. I was thinking of adding bold fonts into echo-s and this might allow me to do that
